src/server/conn/http1.rs

```diff
@@ -359,6 +361,16 @@ impl Builder {
         self
     }
 
+    /// Set whether the `date` header should be included in HTTP responses.
+    ///
+    /// Note that including the `date` header is recommended by RFC 7231.
+    ///
+    /// Default is true.
+    pub fn add_date_header(&mut self, enabled: bool) -> &mut Self {
```

Comment:
I think I'd lean towards making the name `auto_date_header(bool)`, which matches some of the other "auto" options.